recipes/rstudio/meta.yaml

```diff
@@ -0,0 +1,177 @@
+        - {{ cdt('libxxf86vm') }}  # [linux]
+        - {{ cdt('libxext') }}     # [linux]
+      host:
+        - r-base
+        - r-purrr
+        - r-rmarkdown
+        - r-renv
+        - r-testthat
+        - r-xml2
+        - r-yaml
+        - r-rstudioapi
+        - libxkbcommon  # [linux]
+        - xorg-libxcomposite  # [linux]
+        - xorg-libxcursor  # [linux]
+        - xorg-libxscrnsaver  # [linux]
+        - xorg-libxi  # [linux]
+        - xorg-libxtst  # [linux]
+        - openjdk
+        - boost-cpp
+        - soci-postgresql
+        - soci-core
+        - soci-sqlite
+        - libpq
+        - qt
+        - libuuid  # [linux]
+        - bzip2
+        - zlib
+        - pango
+        - sqlite
+        - libcups  # [linux]
+        - openssl
+        - hunspell
+        - hunspell-en
+        - pandoc
+        - mathjax <3.0
+        - yaml-cpp
+        - jpeg  # [osx]
+        - libpng  # [osx]
+      run:
+        - r-base
+        - r-purrr
+        - r-rmarkdown
+        - r-renv
+        - r-testthat
+        - r-xml2
+        - r-yaml
+        - r-rstudioapi
+        - libxkbcommon  # [linux]
+        - libxkbcommon  # [linux]
+        - xorg-libxcomposite  # [linux]
+        - xorg-libxcursor  # [linux]
+        - xorg-libxscrnsaver  # [linux]
+        - xorg-libxi  # [linux]
+        - xorg-libxtst  # [linux]
+        - openjdk
+        - boost-cpp
+        - soci-postgresql
+        - soci-core
+        - soci-sqlite
+        - libpq
+        - qt
+        - libuuid  # [linux]
+        - bzip2
+        - zlib
+        - pango
+        - sqlite
+        - libcups  # [linux]
+        - openssl
+        - hunspell
+        - hunspell-en
+        - pandoc
+        - mathjax <3.0
+        - yaml-cpp
+        - jpeg  # [osx]
+        - libpng  # [osx]
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/rstudio/bin/rstudio  # [linux]
+        - sh -c "xvfb-run -s '-screen 0 1200x800x24' ${PREFIX}/bin/rstudio --no-sandbox --run-diagnostics; cat ~/rstudio-diagnostics/diagnostics-report.txt"  # [linux]
+        - test -f ${PREFIX}/lib/rstudio/RStudio.app/Contents/MacOS/RStudio  # [osx]
+
+about:
+  home: www.rstudio.com/products/rstudio/
+  license: GPL-3.0-or-later
+  license_file: COPYING
+  doc_url: https://github.com/rstudio/rstudio
+  summary: RStudio is an integrated development environment (IDE) for R
+
+about:
+  home: https://www.rstudio.com/
+  license: AGPL-3.0-only
+  license_family: GPL
+  license_file: COPYING
+  summary: RStudio is an integrated development environment (IDE) for R
+  description: |
+    RStudio is an integrated development environment (IDE)
+    for R. It includes a console, syntax-highlighting
+    editor that supports direct code execution, as well
+    as tools for plotting, history, debugging and
+    workspace management.
+  doc_url: https://support.rstudio.com/hc/en-us/sections/200107586-Using-RStudio
+  dev_url: https://github.com/rstudio/rstudio
+
+  
+extra:
```

Comment:
I would add `feedstock-name` here as well, in case you keep this multi outpu.

Reply:
Done in https://github.com/izahn/staged-recipes/commit/380b10ffcb98a5e85dbf3016b85638878c079d0d , thank you!